Assets/Scripts/Robo.cs

```diff
@@ -97,4 +118,18 @@ void FixedUpdate()
         created_duncan.GetComponent<Rigidbody2D>().velocity = displacement * (created_duncan.GetComponent<Duncan>().speed / displacement.magnitude);
         return;
     }
+
+        // Processes damage taken by player
```

Comment:
```suggestion
    // Processes damage taken by player
```

Reply:
oh shoot thanks, I accidentally messed up the spacing while copying stuff over.